buildconfigs/kustomization.yaml

```diff
@@ -0,0 +1,228 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: okd
+
+patches:
+  - patch: |-
+      - op: add
+        path: "/spec/source/images/-"
+        value:  
+           from:
+              kind: "ImageStreamTag"
+              name: "release:builder"
+           as:
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.10'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-etcd-golang-1.16'
+              - 'registry.ci.openshift.org/openshift/release:golang-1.17'
```

Comment:
Not quite sure what this does - adds these images as base to all buildconfigs?

Reply:
Hey @vrutkovs, 

this patch is to bulk replace the (not runtime-stage) FROM layers that link any of the images in the `as` array with the release:builder imagestreamtag.

Currently, we do that for each image in a fine-grained way, but maintaining each single buildconfig "as" stanza in sync with the upstream payload images is harder. I don't know if there is another way like the one we do for the runtime stage base image replacement.

So the patch changes any buildconfig not having the skip label to have this replacement rule. Only the buildconfigs that actually refer to images in the array will consume this. E.g., with the rebase landed yesterday, we will have to change some of the `as` stanza to replace `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.1**9**-openshift-4.12` instead of `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12` (and we also miss golang 1.19 in cs9).

Reply:
It'd be good to add a comment to this file explaining this.

Reply:
Done